src/test/java/com/google/spanner/liquibase/TestHarness.java

```diff
@@ -143,7 +143,7 @@ public static Connection useSpannerEmulator() throws SQLException {
     if (spannerEmulatorHost == null) {
 
       // Create the container
-      final String SPANNER_EMULATOR_IMAGE = "gcr.io/cloud-spanner-emulator/emulator:latest";
+      final String SPANNER_EMULATOR_IMAGE = "gcr.io/cloud-spanner-emulator/emulator:1.3.0";
```

Comment:
The build on CircleCI consistently gets stuck if the `latest` tag is used. Using `1.3.0` (which is currently the latest) seems to work...

Reply:
Nope, does not always work... Reverting to 1.2.0.